pacai/core/featureExtractors.py

```diff
@@ -1,12 +1,60 @@
-# Feature extractors for Pacman game states.
+# Feature extractors for Pacman game states and a private search problem to find
+# the closest food.
 
 import abc
 
 from pacai.core import game
-from pacai.core.search import _ClosestFoodSearchProblem
+from pacai.core.game import Actions
+from pacai.core.game import Directions
+from pacai.core.search.problem import SearchProblem
 from pacai.student import search
 from pacai.util import counter
 
+class _ClosestFoodSearchProblem(SearchProblem):
```

Comment:
Always put top-level privates (classes and functions) after publics (with the exception of main()).